include/seastar/core/memory.hh

```diff
@@ -234,16 +234,19 @@ class statistics {
     size_t _free_memory;
     uint64_t _reclaims;
     uint64_t _large_allocs;
+    uint64_t _failed_allocs;
```

Comment:
Most of the diff is associated with modifying all the places where this additional memory stat needed to be handled.